src/sui-autocompleted/index.jsx

```diff
@@ -205,11 +206,13 @@ Autocompleted.propTypes = {
   initialValue: PropTypes.string,
   placeholder: PropTypes.string,
   selectFirstByDefault: PropTypes.bool,
-  suggests: PropTypes.array.isRequired
+  suggests: PropTypes.array.isRequired,
+  title: PropTypes.string
 }
 
 Autocompleted.defaultProps = {
+  focus: false,
   initialValue: '',
   selectFirstByDefault: true,
-  focus: false
+  title: undefined
```

Comment:
If it's `undefined`, then defaultProp is not needed.

Reply:
fixed